docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+
+## How to debug an SRM
+
+!!! info
+    If you want a TLDR version of this look at the first five items in the list.
+
+Some of this debugging is similar to debugging general software. Is the SRM only on a certain segment of users (i.e. filter by country, OS version, app version, platform, etc). Was there an instrumentation bug that was pushed to a certain app version?
+
+!!! info
+    This list is sorted in rough priority order. 
+
+### 1. Are you using a custom exposure event or the Amplitude exposure event?
+     
+We recommend you use the Amplitude exposure event as this is sent when `fetch()` is called. (TODO AKHIL Should we say something about local event since that doesn't use Amplitude Exposure). On th other hand, if you use a custom exposure event you may not send it at the time the user is exposed to a variant. Note that it does not cost extra money to use Amplitude Exposure events.
+
+### 2. Did they change traffic allocation in the middle of the experiment?
+
+By traffic allocation, I mean changing 50% treatment / 50% control to 60% treatment / 40% control, not changing it from 2% of all traffic going to the experiment to 5% of all traffic going to the experiment. One of the assumptions of the SRM test is that the traffic allocation does not change in the middle of the test. See this [article](https://help.amplitude.com/hc/en-us/articles/7985566141083-Interpret-the-cumulative-exposures-graph-in-Amplitude-Experiment#inflection-point) for why you should not change the traffic allocation in the middle of an experiment. // TODO AKHIL see if can link to specific paragraph.
+
+### 3. Is the analysis time window the whole time the experiment was receiving traffic?
+
+The reason this is important is because if one variant causes users to return to the product then you may see more users than expected in that variant. For example, the experiment ran from January 1 to January 30, and the experiment was analyzed from January 15 to January 30. Say there are 100 users in control and treatment respectively every two weeks. Say the treatment you are experimenting on is so good that it makes the users return the product every day and the control is so bad that they never come back. Then, during the analysis you will have 100 exposed users in control and 200 exposed users in treatment. Here the issue is that you are not doing an apples-to-apples comparison between control and treatment. If the analysis time window and the time the experiment was receiving traffic are different, change the analysis window to be the whole time the experiment was receiving traffic and see if there is still a SRM.
+
+### 4. How much [variant jumping](// TODO AKHIL link to Brian's article) (same user seeing both treatment and control) was there? Is it more than 5%? 
```

Comment:
@bgiori Whenever your variant jumping article is written I can link to it here

Reply:
👍